logrotate/CMakeLists.txt

```diff
@@ -43,20 +60,12 @@ SET(LIBRARY_TO_BUILD g3logrotate)
 
 # Build the library
 INCLUDE_DIRECTORIES(${PROJECT_SRC})
-ADD_LIBRARY(${LIBRARY_TO_BUILD} SHARED ${SRC_FILES})
+ADD_LIBRARY(${LIBRARY_TO_BUILD} ${SRC_FILES})
```

Comment:
Or! Make it an option to use the shared library or the static library. 

See also in the Options.cmake. If you store the library name in a variable then you can keep the logic the same regardless if it is .so or .a library suffix